src/js/modules/components/moveable.js

```diff
@@ -0,0 +1,31 @@
+export const MOVE_END = 'moveend';
+const PREVENT_SELECT_CLASS = 'noselect';
+
+export default function({ view }) {
+  const offset = {};
+  const position = {};
+
+  function onMouseMove({ clientX, clientY }) {
+    position.x = clientX - offset.x;
+    position.y = clientY - offset.y;
+    view.el.style.transform =
+      `translate3d(${position.x}px, ${position.y}px, 0)`;
+  }
+
+  function onMouseUp() {
+    document.removeEventListener('mousemove', onMouseMove);
+    document.removeEventListener('mouseup', onMouseUp, false);
+    view.trigger(MOVE_END, position);
+  }
+
+  function onMouseDown({ clientX, clientY }) {
+    const { left, top } = view.el.getBoundingClientRect();
+    offset.x = clientX - left;
+    offset.y = clientY - top;
+    document.addEventListener('mousemove', onMouseMove, false);
```

Comment:
Because they worked inproperly in case of objects collision. E.g when you dragged object over another and the second object has bigger z-index the first one has stopped - because we lost mouse event. Having global event solves this problem, also this way we are sure that object will never move whenever user releases mouse button.